src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.ts

```diff
@@ -353,6 +360,16 @@ export class InstructorCoursesPageComponent implements OnInit {
     });
   }
 
+  /**
+   * Updates progress bar using ProgressBarService.
+   * This is a helper function used by {@link InstructorCoursesPageComponent#createCourse()} to
+   * reduce code duplication.
+   */
+  private updateProgressBarHandler(copyProgressPercentage: number): void {
+    this.copyProgressPercentage = copyProgressPercentage;
```

Comment:
Actually are we able to just delete `this.copyProgressPercentage` entirely? It seems like it was made solely to be used in `createCourse`

Reply:
Then following that, you can just this `this.progressBarService.updateProgress(this.copyProgressPercentage)` directly without creating `updateProgressBarHandler`

Reply:
Yes, agreed. Thanks! Will make the change.